vm/src/builtins.rs

```diff
@@ -221,7 +221,17 @@ fn builtin_isinstance(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     Ok(vm.context().new_bool(isinstance))
 }
 
-// builtin_issubclass
+fn builtin_issubclass(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    if args.args.len() != 2 {
+        panic!("isinstance(s) expects exactly two parameters");
```

Comment:
Should be issubclass?

Reply:
Yes, but i'll wait for the macro to be merged.